app/_plugins/read_data_file_liquid.rb

```diff
@@ -0,0 +1,66 @@
+# via https://gist.github.com/dgopstein/7fcb514d163f7b090edd5a98b9f3f9a7
+# Treat every _data file as liquid.
+# This allows us to include YAML files in other YAML files.
+require 'tmpdir'
+module Jekyll
```

Comment:
Do we really need to monkey-patch Jekyll?
There's probably a better alternative, what are we trying to solve with this? I'm glad to help

Reply:
This was in place before I inherited the branch.  I definitely defer to you on this. Can you please push a change you recommend?  Thank yoU!